packages/help-center/src/components/utils.tsx

```diff
@@ -40,14 +42,21 @@ export const generateContactOnClickEvent = (
 	}
 };
 
+/**
+ * Returns the last message from a conversation.
+ * @returns The last message or null if there are no messages.
+ */
 export const getLastMessage = ( {
 	conversation,
 }: {
 	conversation: OdieConversation | ZendeskConversation;
-} ) => {
-	return Array.isArray( conversation.messages ) && conversation.messages.length > 0
-		? conversation.messages[ conversation.messages.length - 1 ]
-		: null;
+} ): OdieMessage | ZendeskMessage | null => {
+	const lastMessage =
+		Array.isArray( conversation.messages ) && conversation.messages.length > 0
+			? conversation.messages[ conversation.messages.length - 1 ]
+			: null;
+
+	return lastMessage;
```

Comment:
I believe we could really do without that local variable, it's already pretty clear what that function returns:
```suggestion
	return Array.isArray( conversation.messages ) && conversation.messages.length > 0
		? conversation.messages[ conversation.messages.length - 1 ]
		: null;
```

Reply:
Right, enough information already in the method name.